debezium-quarkus-outbox/integration-tests/src/test/java/io/debezium/outbox/quarkus/it/OutboxOpenTracingTest.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.outbox.quarkus.it;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+/**
+ * Integration tests for the Debezium Outbox extension, enabling OpenTracing behavior by default.
+ *
+ * @author Chris Cranford
+ */
+@QuarkusTest
+@TestProfile(OutboxProfiles.OpenTracing.class)
+public class OutboxOpenTracingTest extends AbstractOutboxTest {
```

Comment:
How is the tracing functionality being tested? Validating a non-null value in the trace id column seems like a reasonable approach?